test/src/org/labkey/test/util/cds/CDSHelper.java

```diff
@@ -178,6 +178,9 @@
     public static final String PROT_Z139 = "z139";
     public static final String PROT_Z140 = "z140";
 
+    public static final String ANTIGEN_HIV_1C = "HIV-1 C.9999.B2 [gp120.D7.avi]";
+
+    public static final String ANTIGEN_HIV_1D = "HIV-1 D.99986.B12 [gx120.D7.avi]";
```

Comment:
Looks like this variable is not being used

Reply:
Created for future use.